pt-BR/mail/chrome/messenger/messengercompose/composeMsgs.properties

```diff
@@ -263,59 +263,59 @@ sendAnyway=Send anyway
 12593=A mensagem não pôde ser enviada através do servidor SMTP %S devido a um motivo que não pôde ser identificado. Por favor, verifique se as configurações do servidor SMTP estão corretas e tente de novo ou contate seu administrador da rede.
 
 ## @name NS_MSG_ERROR_DOING_DRAFTS
-13000=There was an error copying the message to the Drafts folder. Retry?
+13000=Erro ao copiar a mensagem para a pasta Rascunhos. Tentar novamente?
 
 ## @name NS_ERROR_SMTP_AUTH_CHANGE_ENCRYPT_TO_PLAIN_NO_SSL
 # LOCALIZATION NOTE (12594): $S is server hostname
-12594=There was an error copying the message to the Drafts folder. Retry?
+12594=O servidor SMTP %S parece não suportar parece não suportar senhas criptografadas. Se você acabou de configurar esta conta, por favor, tente alterar o 'Método de autenticação' para 'Senha' em 'Configurações de conta' / 'Configurações do servidor'.
 
 ## @name NS_ERROR_SMTP_AUTH_CHANGE_ENCRYPT_TO_PLAIN_SSL
 # LOCALIZATION NOTE (12595): $S is server hostname
-12595=The SMTP server %S does not seem to support encrypted passwords. If you just set up this account, please try changing to 'Normal password' as the 'Authentication method' in the 'Account Settings | Server settings'. If it used to work and now suddenly fails, please contact your email administrator or provider.
+12595=O servidor SMTP %S parece não suportar parece não suportar senhas criptografadas. Se você acabou de configurar esta conta, por favor, tente alterar o 'Método de autenticação' para 'Senha normal' em 'Configurações de conta' / 'Configurações do servidor'. Se antes funcionava e agora parou de funcionar de repente, por favor, entre em contato com o seu administrador ou provedor de e-mails.
 
 ## @name NS_ERROR_SMTP_AUTH_CHANGE_PLAIN_TO_ENCRYPT
 # LOCALIZATION NOTE (12596): $S is server hostname
-12596=The SMTP server %S does not allow plaintext passwords. Please try changing to 'Encrypted password' as the 'Authentication method' in the 'Account Settings | Server settings'.
+12596=O servidor SMTP %S não permite senhas em texto claro. Por favor, tente alterar o 'Método de autenticação' para 'Senha criptografada' em 'Configurações de conta' / 'Configurações do servidor'.
```

Comment:
CORRECTIONS NEEDED it should be 12596=O servidor SMTP %S não permite senhas em texto não criptografado. Por favor, tente alterar o 'Método de autenticação' para 'Senha criptografada' em 'Configurações de conta' / 'Configurações do servidor'.